utilities/infra.py

```diff
@@ -362,7 +362,9 @@ def is_managed_cluster(client: DynamicClient) -> bool:
     for entries in platform_statuses.values():
         if isinstance(entries, kubernetes.dynamic.resource.ResourceField):
             if tags := entries.resourceTags:
-                return next(b["value"] == "true" for b in tags if b["key"] == "red-hat-managed")
```

Comment:
Is the problem because b is `None`? Then would `return next(b["value"] == "true" for b in tags if b and b["key"] == "red-hat-managed")` help?